__tests__/app.test.js

```diff
@@ -63,12 +63,11 @@ describe('2. GET api articles', () => {
                                 body : expect.any(String),
                                 created_at : expect.any(String),
                                 votes : expect.any(Number),
-                                comment_count : expect.any(Number)
+                                comment_count : expect.any(String)
                             })
                         )
                     })
-                    expect(articles[0].article_id).toBe(3);
-                    expect(articles[articles.length - 1].article_id).toBe(7);
+                    expect(articles).toBeSortedBy('created_at', {descending : true});

```

Comment:
toBeSortedBy is great to check sort order, in future you could extract this out to another test. It is good practice when using TDD to have separate tests to test individual behaviours of the function. Not a huge issue though you can still merge 👍